packages/docs/src/examples/v-rating/prop-icon-label.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <v-rating
+    v-model="rating"
+    icon-label="custom icon label text"
```

Comment:
this example does not make use of `{0}`

Reply:
I have added {0} and {1} to the example icon-label to now read `custom icon label text {0} of {1}`.